tests/curl/test_curl_encoding.sh

```diff
@@ -10,15 +10,17 @@ function curl_accept_encoding {
 	local LOG="verbose.log"
 
 	$OSCAP xccdf --verbose=DEVEL eval --fetch-remote-resources --results $RF $DF 2>$LOG || echo "OK"
+	check_command=()
 
 	case $(uname) in
 		FreeBSD)
-			grep -iE "Accept-Encoding.*gzip" $LOG
+			check_command=(grep -iE "Accept-Encoding.*gzip" "$LOG")
 			;;
 		*)
-			grep -iP "Accept-Encoding.*gzip" $LOG
+			check_command=(grep -iP "Accept-Encoding.*gzip" "$LOG")
 			;;
 	esac
+	"${check_command[@]}" || die "Couldnt assert gzip encoding in the log: $(grep -i 'curl header' $LOG)"
```

Comment:
```bash
case ...
   ...
esac || die "Couldnt assert gzip encoding in the log: $(grep -i 'curl header' $LOG)"
```
should be enough, no?

Reply:
And, there is an apostrophe missing.

Reply:
That's cool, I didn't know that one can treat the case block as a command.